### DISCUSSION TITLE
[Test Flight] 문제 공유

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#54] [General]
여러분들이 겪으신 문제를 여기에 적어주세요.

1. 상황이 발생한 시점 (어떤상황인지 구체적이면 좋습니다.)
2. 관련 이미지 or 영상 캡쳐

**Comment by labydin** [comment]
* 다크 모드 관련
1. 탭바 색깔 변경 : 마이페이지 내부 탭에서 희색으로 변경, 이후 지도화면으로 이동해도 흰색 그대로 유지됨
![IMG_3430](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/0092c29d-4df3-43b2-b259-20967863acdf)
2. 텍스트가 시스템 컬러인 경우 보이지 않음
![IMG_3426](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/95fb7ff5-cb40-4993-9a0a-83ff3419487a)
3. 네비게이션 바 컬러 이슈
![IMG_3429](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/3c825102-57e2-4d1f-a0f9-27e8e5b0d3c7)
4. 배경색 통일성 이슈
![IMG_3434](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/de4d2ccb-16c2-4fd0-814e-4fbf0837ac29)

* 지도 서치바 관련
1. 빈화면 터치 시 endediting되어 fetchAPI 작동 -> 실행되는 시점을 return키 터치 시로 바꿔야 할 듯?
![IMG_3427](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/d5804ee4-3ed7-4fe5-8a6b-2825e25c0fd4)
2. '12345' 를 검색하면 외국으로 검색됨 -> 어디까지를 예외 처리할 것인가...
![IMG_3430](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/3ec11b87-f25e-47a3-bb98-fff065e195c4)

* 커뮤니티 키보드 관련
1. 키보드 올라오면 대화 내용이 가려지는 문제
![IMG_3438](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/c9a05bde-0bdf-4e6c-a789-28e1621655a1)

* 프로필 수정 관련
1. 프로필 수정 완료 버튼 클릭 후 무한로딩(근데 이건 딱 한 번만 이래서 이 문제가 발생하는 정확한 타이밍을 잘 모르겠습니다..)
![IMG_3435](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/6d4f4538-71bc-4421-b22b-8e8d068f77d3)

**Comment by labydin** [comment]
이건 궁금해서 한 번 해봤어요 ㅎㅎ
![IMG_3431](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/49cbeb96-d669-4121-8970-569f1e8ad717)

**Comment by jinmoon23** [comment]
![IMG_8165](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/161567425/97fad93e-32af-4b36-82a7-b324216819c0)
검색 -> 앱 전환 -> 다시 돌아온 경우 떡볶이 어노테이션 사라짐

![IMG_8167](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/161567425/6c80fccf-a7a1-4e9b-9660-8474af33505c)
앱 최초 구동 시 내 주변 분식집 표시되지 않는게 정상인가요?!

![IMG_8168](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/161567425/b6c64e8a-592b-4dcb-b141-25e7a5ec723d)
title을 받아올 때 numberOfLines 를 2로 해야할 것 같습니다! 그리고 북마크 클릭 시 해당하는 카드의 DetailVC로 이동하면 좋을 것 같습니다. 
추가적으로 해당 페이지에서 북마크 버튼을 탭하면 커스텀얼럿이 뜨고 FB 연동하여 북마크를 해제할 수 있도록 하면 좋을 것 같습니다.

**Reply by jinmoon23** [reply]
추가적으로 마이페이지 스와이프해서 뒤로가기 구현하면 좋을 것 같아요!

**Comment by labydin** [comment]
저도 마이페이지 화면에서 스크랩된 가게 이름이나 북마크 카드를 클릭하면 디테일 화면으로 넘어가면 좋겠다고 생각했는데, 이게 back 버튼 누르면 dismiss가 마이페이지로 가게 할 수 있나요?? 아님 분기 처리해야 하나요??

**Reply by jinmoon23** [reply]
디테일 화면이라고 하시면 Map에서 어노테이션핀을 클릭하고 들어간 페이지를 말씀하시는 거죠??
그렇다면 마이페이지에서 dismiss로는 안되지 않을까요?? dismiss는 직전의 화면으로 가지 않을까 싶습니다! 
추측컨데 탭바이동을 하고 검색을 하고 핀을 누르고 디테일까지 접근을... 하기에는 너무 복잡하네요.. dismiss 만으로 가능하면 좋겠긴 하네요..ㅠㅠ

**Comment by Haroldfromk** [comment]
![IMG_D321B7CA385B-1](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/97341336/dc26d6b6-d895-4de7-878a-bdc1b18bea3a)

리뷰 작성시 아무것도 적히지 않았을때를 대비한 Exception Handling 필요

**Comment by Haroldfromk** [comment]
애플계정 로그아웃 시 한번에 안되는 문제.

**Reply by Haroldfromk** [reply]
시뮬레이터 및 기기로 띄웠을때도 정상 로그아웃 되는것 확인

**Comment by jinmoon23** [comment]
https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/161567425/850f46f6-ae32-4eeb-8552-ffe45a88d89f

검색 후 x누르면 주소읽어올 수 없다는 얼럿메세지 발생

**Comment by labydin** [comment]
![IMG_EF878E863E12-1](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/1080cbf3-2967-4242-931b-8803b1832348)
![IMG_A32B934C4FF3-1](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/22579901-c0cd-4aa1-af26-4c8625de8d16)
다크모드에서 스크롤 내리면 Navigation bar 회색으로 변경

**Comment by labydin** [comment]
![IMG_3453](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/eb21cb7d-cb11-4803-8eb3-f0f382866ec8)
다크모드에서 앱타이틀 보이지 않음

**Comment by labydin** [comment]
앱이 foreground로 올라올 때 추천 화면 무한 로딩
